src/common/pbconvert/tests/CMakeLists.txt

```diff
@@ -14,7 +14,15 @@ set(TARGET_NAME pbconvert_test)
 # Sources
 # ######################################################################################################################
 
-set(SOURCES common.cpp iam.cpp sm.cpp)
+set(SOURCES common.cpp)
+
+if(WITH_IAM_API)
```

Comment:
We have local WITH_IAM and WITH_SM options. Use them.

Reply:
Then the test will not work for MP